document_url/static/src/js/url.js

```diff
@@ -13,13 +13,14 @@ odoo.define('document_url', function (require) {
         events: _.extend(AttachmentBox.prototype.events, {
             "click span.o_add_url_button": "_onAddUrl",
         }),
+
         /**
          * Opens wizard to add an URL attachment to the current record
          *
          * @private
          * @param {MouseEvent} ev
          */
-        _onAddUrl: function (ev) {
```

Comment:
This file is from another module, it shouldn't change on this PR

Reply:
@ivantodorovich These changes I ended up changing in this PR because it was failing Travis. But I can do it separately. What do you think ?

Reply:
@ivantodorovich I removed the change in the document_url module